pkg/controller/bypass/bypass_controller.go

```diff
@@ -153,6 +146,14 @@ func addIptables(ns string) error {
 
 	execFunc := func(netns.NetNS) error {
 		log.Infof("Running add iptables rule in namespace:%s", ns)
+		//To avoid iptables rules being added multiple times due to problems with k8s resource synchronization
+		delIptArgs := [][]string{
+			{"-t", "nat", "-D", "PREROUTING", "-j", "RETURN"},
```

Comment:
Is this safe if the no such rule exist

Reply:
> Is this safe if the no such rule exist

yes, when this rule is not present, an error will occur, but it will not be thrown out in the log and will have no effect on the function, so you can just ignore it.